apt-requirements.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Keep sorted.
+autoconf
+bison
+build-essential
+clang-format
+curl
+flex
+g++
+git
+libelf1
```

Comment:
libelf-dev is required on the Verilator build machines, it's OK to always install it. 

(That's why CI is failing at the moment.)

Reply:
Fixed.

Reply:
Done, merging.